content/en/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,70 @@
+---
+reviewers:
+- dchen1107
+- roberthbailey
+- liggitt
+title: Control plane-Node Communication
```

Comment:
Hello @Fale .
To recap, this pull request:
- renames the `master-node-communication.md` file to `control-plane-node-communication.md`
- changes the title of the page
- removes the term `master` from the file and replaces with `control plane`
- removes other related text

In the future, you may want to make the page edits first in a commit before renaming the file so that reviewers can compare the changes.

I know there is some question about capitalizing the page titles. In this case, should `plane` be capitalized?

Reply:
I'd capitalize “Plane”. If you do split this into 2 commits @Fale that'd be handy (also helps people rebase existing work if need be).